src/Parser/Hunter/BeastMastery/Modules/Abilities.js

```diff
@@ -117,7 +119,7 @@ class Abilities extends CoreAbilities {
       },
       {
         spell: SPELLS.BARRAGE_TALENT,
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
```

Comment:
Since it has a cast efficiency, I'm assuming it is used in single target? If so I think it's better to keep it as rotational. Otherwise I think the cast efficiency should be removed (afaik we don't show cast efficiency for AOE only spells, but I could be wrong).